src/main/webapp/components/query-manager/query-manager.stories.tsx

```diff
@@ -12,7 +12,7 @@ import { button } from '@storybook/addon-knobs'
 
 stories.add('Basic', () => {
   const [queries, setQueries] = useState(sampleQueries)
-  const [query, setQuery] = useState(sampleQueries[0].id)
+  const [query, setQuery] = React.useState(sampleQueries[0].id)
```

Comment:
:pencil2: unnecessary change

Reply:
I didn't want this action to trigger when selecting a query https://github.com/connexta/revelio/blob/master/src/main/webapp/@storybook/use-state.js#L8